website_sale_cart_selectable/views/product_view.xml

```diff
@@ -1,17 +1,15 @@
 <?xml version="1.0"?>
-<openerp>
-    <data>
+<odoo>
 
         <record model="ir.ui.view" id="res_product_view_form">
             <field name="name">Product Add To Cart</field>
             <field name="model">product.template</field>
             <field name="inherit_id" ref="website_sale.product_template_form_view"/>
             <field name="arch" type="xml">
-                <field name="website_published" class="pull-right" widget="website_button" position="after">
-                    <field name="website_btn_addtocart_published" class="pull-right" widget="website_button"/>
+                <field name="public_categ_ids" position="after">
+                    <field name="website_btn_addtocart_published"/>
```

Comment:
@pedrobaeza That widget is for stat buttons now. I could put this configuration in the button box, but I thought it belonged to the place where it used to be. I don't know if there is in v10 a widget similar to that one, do you?